code/modules/antagonists/eldritch_cult/knowledge.dm/flesh_lore.dm

```diff
@@ -0,0 +1,257 @@
+	next_knowledge = list(/datum/eldritch_knowledge/summon/stalker,/datum/eldritch_knowledge/spell/rust_wave)
+
+/datum/eldritch_knowledge/summon/rusty
+	name = "Rusted Ritual"
+	gain_text = "I combined principle of hunger with desire of corruption. The rusted hills call my name."
+	desc = "You can now summon a Rust Walker transmutating vomit pool, a head and a book."
+	cost = 1
+	required_atoms = list(/obj/effect/decal/cleanable/vomit,/obj/item/bodypart/head,/obj/item/book)
+	mob_to_summon = /mob/living/simple_animal/hostile/eldritch/rust_spirit
+	next_knowledge = list(/datum/eldritch_knowledge/summon/stalker,/datum/eldritch_knowledge/spell/mad_touch)
+
+/datum/eldritch_knowledge/spell/blood_siphon
+	name = "Blood Siphon"
+	gain_text = "Our blood is all the same after all, the owl told me."
+	desc = "You gain a spell that drains enemies health and restores yours."
+	cost = 1
+	spell_to_add = /obj/effect/proc_holder/spell/targeted/touch/ash_leech
+	next_knowledge = list(/datum/eldritch_knowledge/spell/rust_wave,/datum/eldritch_knowledge/spell/mad_touch)
+
+/datum/eldritch_knowledge/flesh_final
+	name = "Priest's Final Hymn"
+	gain_text = "Man of this world. Hear me! For the time of the lord of arms has come!"
+	desc = "Bring 3 bodies onto a transmutation rune to either ascend as a terror of the night prime or you can summon a regular terror of the night."
+	required_atoms = list(/mob/living/carbon/human)
+	cost = 3
+	route = "Flesh"
+	var/is_summoned = FALSE
+
+/datum/eldritch_knowledge/flesh_final/recipe_snowflake_check(list/atoms, loc)
+	if(is_summoned)
+		return FALSE
+	var/counter = 0
+	for(var/mob/living/carbon/human/H in atoms)
+		counter++
+	if(counter >= 3)
+		return TRUE
+	return FALSE
+
+/datum/eldritch_knowledge/flesh_final/on_finished_recipe(mob/living/user, list/atoms, loc)
+	var/alert_ = alert("Do you want to ascend as the lord of the night or just summon a terror of the night?","...","Yes","No")
+	switch(alert_)
+		if("No")
+			var/mob/living/summoned = new /mob/living/simple_animal/hostile/eldritch/armsy(loc)
+			message_admins("[summoned.name] is being summoned by [user.real_name] in [loc]")
+			var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a [summoned.real_name]", ROLE_ECULTIST, null, ROLE_ECULTIST, 50,summoned)
+			if(!LAZYLEN(candidates))
+				return
+			var/mob/dead/observer/C = pick(candidates)
+			message_admins("[key_name_admin(C)] has taken control of ([key_name_admin(summoned)]).")
+			summoned.ghostize(0)
+			summoned.key = C.key
+			is_summoned = TRUE
+
+			to_chat(summoned,"<span class='warning'>You are bound to [user.real_name]'s' will! Don't let your master die, protect him at all cost!</span>")
+		if("Yes")
+			var/mob/living/summoned = new /mob/living/simple_animal/hostile/eldritch/armsy/prime(loc,spawn_more = TRUE,len = 10)
+			summoned.ghostize(0)
+			var/mob/living/carbon/C = user
+			summoned.key = C.key
```

Comment:
can't you just move their mind into the summoned prime armsy, so that they can keep their objectives and stuff?

Reply:
another good catch, i ll try to do that